Dockerfile

```
@@ -2,12 +2,12 @@
 
 # This enforces that the packages downloaded from the repositories are the same
 # for the defined date, no matter when the image is built.
-ARG NOBLE_DATE=20240801
-ARG APT_UPDATE_SNAPSHOT=${NOBLE_DATE}T030400Z
+ARG UBUNTU_TAG=noble-20240827.1
```

```suggestion
ARG UBUNTU_TAG=noble-20240904.1
```

It's about to be released, let's keep an eye on [this PR](https://github.com/docker-library/official-images/pull/17652)

---

We can bump this on a later PR, once released.